ci/pipelines/toolbox-staging-deploy.yml

```diff
@@ -46,6 +46,11 @@ resource_types:
       username: ((docker-username))
       password: ((docker-password))
       tag: dev
+  - name: registry-image-resource-1-1-0
+    type: registry-image
+    source:
+      repository: concourse/registry-image-resource
+      tag: "1.1.0"
```

Comment:
Does this version work with the AWS role switching?

Reply:
I'm not sure. Would you like me to try? Is it to do with https://github.com/alphagov/pay-ci/blob/master/ci/pipelines/toolbox-staging-deploy.yml#L102-L105?

Reply:
No, not quite - it's to allow the Concourse worker to assume the `aws_role_arn`. If it's possible to use a versioned resource type instead of our `dev-registry-image` resource type (line 42 above)  that's preferable, but I didn't think it had been released in the stable version yet?

Reply:
Oh, it looks like it works:
```
  - name: toolbox-ecr-registry-staging
    type: registry-image-resource-1-1-0
    icon: docker
    source:
      repository: govukpay/toolbox
      aws_access_key_id: ((readonly_access_key_id))
      aws_secret_access_key: ((readonly_secret_access_key))
      aws_session_token: ((readonly_session_token))
      aws_role_arn: arn:aws:iam::((pay_aws_staging_account_id)):role/concourse
      aws_ecr_registry_id: "((pay_aws_staging_account_id))"
      aws_region: eu-west-1
      variant: release
```
There's a green build resulting from this: https://cd.gds-reliability.engineering/teams/pay-deploy/pipelines/toolbox-staging-deploy/jobs/toolbox-staging-deploy/builds/7

Reply:
I'll delete the `dev-registry-image` as it's not being used anyway.

Reply:
Deleted